docs/topic_guides/evaluation/clusters/graph_metrics.md

```diff
@@ -0,0 +1,91 @@
+
+!!! note
+
+    It is important to bear in mind that whilst graph metrics can be very useful for assessing linkage quality, they are rarely definitive, especially when taken in isolation. A more comprehensive picture can be built by considering various metrics in conjunction with one another.
+
+    It is also important to consider metrics within the context of their distribution and the underlying dataset. For example: a cluster density (see below) of 0.4 might seem low but could actually be above average for the dataset in question; a cluster of size 80 might be suspiciously large for one dataset but not for another.
+
+
+## ⚫️ Node metrics
+
+Node metrics quantify the properties of the nodes which live within clusters.
+
+### Example: node degree
+
+Node degree is the number of edges connected to a node.
+
+High node degree is generally considered good as it means there are many edges in support of records in a cluster being linked. Nodes with low node degree could indicate links being missed (false negatives).
+
+However, erroneous links (false positives) could also be the reason for high node degree, so it can be useful to validate the edges of highly connected nodes.
+
+It is important to consider [cluster size]() when looking at node degree. By definition, larger clusters contain more nodes to form links between, allowing nodes within them to attain higher degrees compared to those in smaller clusters. Consequently, low node degree within larger clusters can carry greater significance.
+
+Bear in mind, that the degree of a single node in a cluster isn't necessarily representative of the overall connectedness of a cluster. This is where [cluster centralisation]() can help.
+
+## 🔗 Edge metrics
+
+Edge metrics quantify the properties of the edges within a cluster. 
+
+### Example: 'is bridge'
+
+An edge is classified as a 'bridge' if its removal splits a cluster into two smaller clusters.
+
+[insert picture]
+
+Bridges can be signalers of false positives in linked data, especially when joining two highly connected sub-clusters. Examining bridges can shed light on potential errors in the linking process leading to the formation of false positive links.
+
+## :fontawesome-solid-circle-nodes: Cluster metrics
+
+Cluster metrics refer to the characteristics of a cluster as a whole, rather than the individual nodes and edges it contains.
+
+### Example: cluster size
+
+Cluster size refers to the number of nodes within a cluster.
+
+When thinking about cluster size, it is important to consider the size of the biggest clusters produced and ask yourself, does this seem reasonable for the dataset being linked? For example, does it make sense that one person is appearing hundreds of times in the linked data resulting in a cluster of over 100 nodes? If the answer is no, then false positives links are probably being formed. This could be, for example, due to having blocking rules which are too loose.
```

Comment:
I feel like maybe the focus for false positives should be on comparisons that are giving undue weight to certain bits of evidence, rather than the blocking rules being loose. I think usually we wouldn't use blocking rules to try to exclude false positives - if we had the resources we would probably not block at all

Reply:
Thanks Andy, this is a really good point!